ihontaryk/homework_8/warrior.py

```diff
@@ -0,0 +1,58 @@
+from logging import getLogger
+import logging
+
+
+class Player:
+    def __init__(self, health=50, attack=5):
```

Comment:
why do you need input parameters if you never use them, and actually you should not use them, since unit is created with default values